packages/gatsby-source-shopify/README.md

```diff
@@ -70,6 +70,29 @@ plugins: [
       // Possible values are: 'shop' and 'content'.
       // Defaults to ['shop', 'content'].
       includeCollections: ["shop", "content"],
+
+      // Allow overriding the default queries
+      // This allows you to include/exclude extra fields when sourcing nodes
+      // Available keys are: articles, blogs, collections, products, shopPolicies, and pages
+      // Queries need to accept arguments for first and after
+      // Defaults can be found in ./src/queries.js
```

Comment:
Users can read this README on npm, gatsbyjs.org or github - `./src/queries.js` will only be helpful on github. Additionally that file doesn't contain the keys for `shopifyQueries` map, just query string values.

It almost feel like we need to include default queries so this README works as standalone. Not great solution but maybe we could have some appendix at the end of this file and link to it from here?